src/custom/pages/Profile/index.tsx

```diff
@@ -45,6 +46,15 @@ export default function Profile() {
   const hasOrders = useHasOrders(account)
 
   const vCowBalance = useTokenBalance(account || undefined, chainId ? V_COW[chainId] : undefined)
+  const cowBalance = useTokenBalance(account || undefined, chainId ? COW[chainId] : undefined)
+
+  const { vested, total, unvested } = useVCowData()
+
+  // SHOULD BE REMOVED BEFORE MERGE
+  console.force.log('vested', vested?.toString())
+  console.force.log('total', total?.toString())
+  console.force.log('unvested', unvested?.toString())
+  console.force.log('cowBalance', formatMax(cowBalance, cowBalance?.currency.decimals))
```

Comment:
Do we need to force now to do a log @W3stside? I thought the force is a scape hatch, but not necessary for local. Or this is for letting us see it in the PRaul?

Reply:
its just for PRaul rn since i haven't had a chance to make a PR check environments more granularly than "production"